flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java

```diff
@@ -73,15 +73,15 @@ protected TableEnvironment getTableEnv() {
   }
 
   protected static TableResult exec(TableEnvironment env, String query, Object... args) {
-    return env.executeSql(String.format(query, args));
+    return env.executeSql(args.length > 0 ? String.format(query, args) : query);
   }
 
   protected TableResult exec(String query, Object... args) {
     return exec(getTableEnv(), query, args);
   }
 
   protected List<Object[]> sql(String query, Object... args) {
-    TableResult tableResult = exec(String.format(query, args));
+    TableResult tableResult = exec(query, args);
```

Comment:
I think that this change is correct, but I don't see why it was necessary in this commit. Wouldn't all of the tests run the same way without modifying this file at all?

Reply:
the `LIKE` sql in test case will call this method, I think it is maybe a small mistake, so I update it   to pass the original parameters `query, args`, in the final method `exec(TableEnvironment env, String query, Object... args)` to do the format

Reply:
I remove the ` return env.executeSql(args.length > 0 ? String.format(query, args) : query);` from `exec` method, but if we retain the `String.format` in `sql(String query, Object... args)` method ,the sql will still be formatted twice，even though we use the `%%` , the sql also is formated failed